templates/_formhelpers.html

```diff
@@ -0,0 +1,12 @@
+{% macro render_field(field) %}
```

Comment:
make this take advantage of bootstrap form styling: http://getbootstrap.com/components/#input-groups